adabot/lib/circuitpython_library_validators.py

```diff
@@ -314,8 +317,8 @@ def _validate_readme(self, repo, download_url):
         if "Documentation Status" not in badges:
             errors.append(ERROR_README_MISSING_RTD_BADGE)
 
-        if "Build Status" not in badges:
-            errors.append(ERROR_README_MISSING_TRAVIS_BADGE)
+        #if "Build Status" not in badges:
```

Comment:
hiya can you change this to check for action badge? see https://github.com/adafruit/Adafruit_CircuitPython_RGBLED/blob/master/README.rst for example

Reply:
Done. Added new log below..